groupsforsections.py

```diff
@@ -0,0 +1,85 @@
+import json
+import sys
+import logging
+import requests
+import urllib
+import utils
+import yaml
+
+
+def create_group_category():
+    logging.debug('create_group_category() Called')
+
+    url = canvas_url + '/api/v1/courses/' + course_id + '/group_categories?name=' + urllib.quote_plus(
+            group_category_name)
+    response = api_handler(url, 'Post')
```

Comment:
`'Post'` here (and below, along with `'Get'`) should be a constant, not a string.  See whether Python already has constants defined for HTTP methods.  If not, define `HTTP_METHOD_POST = 'POST'`, etc. yourself.